frontend/src/components/CourseDescriptionPanel/CourseAttributes/CourseAttributes.tsx

```diff
@@ -15,10 +15,52 @@ type CourseAttributesProps = {
   courseCapacity?: EnrolmentCapacityData;
 };
 
+interface UnilectivesCourse {
+  courseCode: string;
+  archived: boolean;
+  // attributes: any[];
+  calendar: string;
+  campus: string;
+  description: string;
+  enrolmentRules: string;
+  equivalents: string[];
+  exclusions: string[];
+  faculty: string;
+  fieldOfEducation: string;
+  genEd: boolean;
+  level: number;
+  school: string;
+  studyLevel: string;
+  terms: number[];
+  title: string;
+  uoc: number;
+  overallRating: number;
+  manageability: number;
+  usefulness: number;
+  enjoyability: number;
+  reviewCount: number;
+}
+
+const getCourseRating = async (code: string) => {
+  const res = await fetch(`https://unilectives.devsoc.app/api/v1/course/${code}`);
```

Comment:
could we use axios instead of the fetchapi here?

Reply:
and potentially move this and its response type out to the api folder?

Reply:
Fixed